ExomeDepth/make_BEDdetail.py

```diff
@@ -230,16 +239,22 @@ def calculate_statistics(data, status):
         'ratio_stdev_{0}'.format(status): "n/a",
         'bfs_{0}'.format(status): "n/a",
         'ratios_{0}'.format(status): "n/a",
+        'refsets_{0}'.format(status): "n/a"
```

Comment:
before it ended with a comma

Reply:
Do you refer to each item with list "stats", or to a different list?
The comma in this list is not needed for the last item.